src/vs/workbench/browser/parts/statusbar/media/statusbarpart.css

```diff
@@ -5,6 +5,7 @@
 
 .monaco-workbench > .part.statusbar {
 	box-sizing: border-box;
+	cursor: default;
```

Comment:
@LeonardoBraga can we just fix it for this particular issue and apply a `cursor:default` style for the progress message [here](https://github.com/Microsoft/vscode/blob/777c8e57e0eb1cbffb87a57ffd4f9c3d274f9429/src/vs/workbench/services/progress/browser/progressService2.ts#L50)?

Reply:
Definitely. My reasoning behind setting the default cursor style instead of specializing each for each status bar part is that - unless required otherwise - all items in the status bar should not have a "selectable" cursor or a "clickable" one. If you don't agree with this reasoning just let me know and I'll apply the rule directly to the progress items.

Reply:
@LeonardoBraga I think the only one that is not working OK right now is the progress one, all the others should get the cursor: default if they are using spans.

Reply:
@bpasero unless I'm missing something, in `statusbarpart.css`, the only selector that sets the cursor to `default` is `.monaco-workbench > .part.statusbar > .statusbar-entry > span`, which would not match `.statusbar-item` and even for `.statusbar-entry`, it would only match the first span.

Besides, having the `cursor: default` in the selector I suggested would even allow us to remove it from `.statusbar-entry > span`. Do you still prefer to proceed with the specific selector added to the progress item?

Reply:
@LeonardoBraga yeah fair enough.